src/test/frank-single-transaction.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# bash strict mode
+set -euo pipefail
+IFS=$'\n\t'
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+cd "${SCRIPT_DIR}/../../"
+
+# create test configuration for fddev
+TMPDIR=$(mktemp -d)
+cat > ${TMPDIR}/config.toml <<EOM
+[development]
```

Comment:
Might be better to have the actual file in the test dir?

Reply:
The underlying motivation is to explicitly guarantee the configuration at runtime, if we can do that another way let's do it.